Overview page updates

JMoore5353 opened the pull request:
The organization of the User Guide section felt a little confusing to me, so I refactored it a little bit. Let me know what you think.

I added a ROSplane overview page in the User Guide, since we had no real landing page that described what ROSplane *was*. On this same note, I added a ROSplane overview to the developer guide, since we didn't have a landing page there. The idea with these landing pages is that it helps people who visit the site know where to start and organizes the rest of the sections they are a part of.

bsutherland333 commented:
I agree that the organization is a bit confusing, having different sections for ROSflight, ROSplane, and ROScopter is a good idea. I don't know if I love calling the ROSflight section "Fly with ROSflight" though, what if we just called it "ROSflight" and then made sure the user guide overview is very clear about the roles ROSflight, ROSplane, and ROScopter?

JMoore5353 commented:
> I agree that the organization is a bit confusing, having different sections for ROSflight, ROSplane, and ROScopter is a good idea. I don't know if I love calling the ROSflight section "Fly with ROSflight" though, what if we just called it "ROSflight" and then made sure the user guide overview is very clear about the roles ROSflight, ROSplane, and ROScopter?

That makes sense. I updated that to ROSflight and refactored some of the information I felt like belonged in other sections to those sections (RC -> RC page, LEDs -> board page). That way the text just describes the main components of ROSflight as seen in the figure, and also links to more information.

How do you guys feel about the `tldr` at the top? :) The goal is to make it as clear as possible where people need to get started... I keep visiting research project sites that have no explanation of what they are / what is going on with the software, so I'd like that to be clear here.
